### DISCUSSION TITLE
New Farming System, How to init the keys.

### DISCUSSION THREAD

**Discussion by JoJ123** [Chia-Network/chia-blockchain#5207] [Support]
I had my gaming machine as the plotter and a docker maching as a farmer.

Now I setup a VM Ubuntu Server as the Farming Machine. I will stop the docker one.

What should I do, that my new VM can farm the same plots?
Is there a way to run chia init with my 24 words? Will this then work with the same plots?

Best,
Johannes

**Comment by lvcivs** [comment]
From the CLI, use `chia keys add` to input your mnemonics. This will add the same set of keys to your VM server that you have been using. You can then use `chia keys show` to verify. It will work with the same plots.

**Reply by gcraenen** [reply]
Coping over the contents of the DB directory will keep from having to sync from scratch. And also copy over the wallet db too.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
